cdisc_rules_engine/models/dataset/pandas_dataset.py

```diff
@@ -38,6 +38,14 @@ def groups(self):
     def empty(self):
         return self._data.empty
 
+    @property
+    def at(self):
```

Comment:
I think we should have some logic that will accomodate this in the dask_dataset.py too. In case dask is used to handle the dataset, it may give an error.